src/Util/PrintInfo.cs

```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+
+namespace sodoff.Utils;
+
+public class PrintInfo {

```

Comment:
why separate class for this? why not use `VerboseLogging` or just direct use `Console.WriteLine` instead of  `PrintInfo.print();` in `Program.cs`?

Reply:
i have put all 3 files in one